scripts/src/Program/Ldd.hs

```diff
@@ -26,41 +19,42 @@ instance Program Ldd where
 type Entry = (String, String, String)
 
 -- String that separates .so name and its absolute path
-arrow = " => " :: TL.Text
+arrow :: Text
+arrow = " => "
 
 -- eg. called with "libgraphite2.so.3 => /lib64/libgraphite2.so.3"
 -- returns ("libgraphite2.so.3", "/lib64/libgraphite2.so.3")
 -- but if no " => " is present, then single path is assumed and returned
 -- as first (relative path) or second element (absolute path)
 -- Note: the point is not to get tricked with irregular vdso and ld.so entries.
-parseNameAndPath :: TL.Text -> (TL.Text, TL.Text)
-parseNameAndPath (TL.strip -> input) =
-        if TL.null afterArrow then fromSingle else (beforeArrow, afterArrow)
+parseNameAndPath :: Text -> (Text, Text)
```

Comment:
I’d like to see these parsing functions use an actual parser combinator library. It’d make the code significantly clearer, and also improve safety. I suggest `megaparsec` as it’s the one we use in Luna.